From 0e39da6d5241aefd07bd194d12d0a54282be5c3b Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 6 Mar 2025 10:12:20 +0100
Subject: [PATCH] arm64: dts: add 3v3 regulator to dmics on stm32mp257f-dk

Add 3v3 regulator to digital microphone nodes on STM32MP257F-DK board.

Change-Id: Id266af760a236d6fa2bccd968efb9282ca3b026e
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/442243
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -59,6 +59,7 @@
 		compatible = "dmic-codec";
 		#sound-dai-cells = <1>;
 		sound-name-prefix = "dmic0";
+		vref-supply = <&scmi_v3v3>;
 		status = "okay";
 
 		port {
@@ -72,6 +73,7 @@
 		compatible = "dmic-codec";
 		#sound-dai-cells = <1>;
 		sound-name-prefix = "dmic1";
+		vref-supply = <&scmi_v3v3>;
 		status = "okay";
 
 		port {
